exercises/concept/kitchen-calculator/.docs/hints.md

```diff
@@ -4,12 +4,11 @@
 
 - [Tuples][tuple-module] are data structures which are arranged in contiguous memory and can hold any data-type.
 - Atoms may be used to denote finite states, as this exercise uses `:cup`, `:fluid_ounce`, `:teaspoon`, `:tablespoon`, `:milliliter` to denote the units used.
-- You may use `Kernel` or `Tuple` functions or pattern matching to manipulate to manipulate the tuples.
+- You may use [`Kernel`][elem] or [`Tuple`][tuple-module] functions or pattern matching to manipulate the tuples.
 
 ## 1. Get the numeric component from a volume-pair
 
-- Consider using a `Kernel` module function to return the volume-pair's numeric component.
-- Remember, one-line functions are best used for short Elixir functions.
```

Comment:
The statement is true, but it doesn't help a student that is struggling with the exercise.